### PR TITLE
fix: response typecast issue

### PR DESCRIPTION
**Pull request by ankitdas13**
resolved note type [issue](https://github.com/razorpay/razorpay-php/issues/15) .

Screenshot 2022-09-27 at 6 21 18 PM (1)

### CODE REVIEW

File: src/Entity.php [hunk 1]
```diff
@@ -192,7 +192,9 @@ public function fill($data)
                 }
                 else
                 {
-                    $value = static::buildEntity($value);
+                    $stdCls = new StdClass();
```
Comment (ramth05) [inline src/Entity.php:195]: @ankitdas13 , why the fix going outside? can't we place it `static::buildEntity`
Also looks like we are missing tests here.

Comment (ankitdas13) [reply src/Entity.php:195]: @ramth05 We are only replacing Entity (Payment,Order,Subscriptions etc..) instance to `StdObject` for associative array and `static:buildEntity` was there before but while fixing this [bug](https://github.com/razorpay/razorpay-php/issues/15) (raised by nemo) i only got this solution if you have any approach then please suggest me.

File: src/Entity.php [hunk 2]
```diff
@@ -146,11 +146,22 @@ protected static function getDefinedEntitiesArray()
...
+            'transfer',
+            'qr_code',
+            'virtual_account',
+            'payment_link',
             'settlement');
     }
 
     protected static function getEntityClass($name)
     {
+        $name = str_replace('_', '', ucwords($name, '_'));
```
Comment (ankitdas13) [inline src/Entity.php:164]: handle entities like `qr_code` & `virtual_account`

File: src/Entity.php [hunk 3]
```diff
@@ -146,11 +146,22 @@ protected static function getDefinedEntitiesArray()
...
+            'items',
+            'plan',
+            'subscription',
+            'addon',
+            'invoice',
+            'qrcode',
+            'transfer',
+            'qr_code',
+            'virtual_account',
+            'payment_link',
```
Comment (ankitdas13) [inline src/Entity.php:158]: added missing entities

**Review by ndhaka007 — APPROVED** [review]
LGTM

**Pull request merged**